### PR TITLE
metrics: add grafana and alertmanager scripts

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
add binlog's grafana and alertmanager scripts, include kafka

### What is changed and how it works?

cp from tidb-ansible

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@july2993 @suzaku PTAL

File: metrics/grafana/reparo.json [hunk 1]
```diff
@@ -0,0 +1,609 @@
+{
```
Comment (july2993) [inline metrics/grafana/reparo.json]: what the hell is this file? I can't find any related metrics, I think it's never used.

Comment (WangXiangUSTC) [reply metrics/grafana/reparo.json]: cp from https://github.com/pingcap/tidb-ansible/blob/master/scripts/reparo.json

Comment (WangXiangUSTC) [reply metrics/grafana/reparo.json]: seems is uselss, I will delete it

Comment (july2993) [reply metrics/grafana/reparo.json]: how about also remove the ones about kafka since it's not related to this repo(like downstream tidb).

Comment (WangXiangUSTC) [reply metrics/grafana/reparo.json]: ok

**Pull request merged**